gpt_researcher.py

```
@@ -0,0 +1,701 @@
+            Given Abstract: {abstract}
+'''),
+    ]
+
+    chat = ChatOpenAI(model="gpt-3.5-turbo")
+
+    return chat.batch([messages])[0].content
+
+
+
+
+from langchain.schema import SystemMessage, HumanMessage
```

Ruff (0.8.2)

Redefinition of unused `SystemMessage` from line 7

(F811)